java/vector/src/main/java/org/apache/arrow/vector/ipc/JsonFileReader.java

```diff
@@ -564,8 +564,10 @@ private ArrowBuf readIntoBuffer(BufferAllocator allocator, BufferType bufferType
     } else if (bufferType.equals(OFFSET)) {
       if (type == Types.MinorType.LARGEVARCHAR || type == Types.MinorType.LARGEVARBINARY) {
```

Comment:
could you add a check for `Types.MinorType.LARGELIST` here are remove the changes below?

Reply:
you also won't need `typeBitWidth` as an arg

Reply:
thanks, done